cms/djangoapps/contentstore/management/commands/clean_cert_name.py

```diff
@@ -0,0 +1,173 @@
+from django.core.management.base import BaseCommand, CommandError
+from collections import namedtuple
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore import ModuleStoreEnum
```

Comment:
Nitpick: the most recent version of Pylint would flag this block of imports! Please separate standard library, third-party, and first-party imports into separate groups.